pandas/tests/tools/test_to_datetime.py

```diff
@@ -1862,6 +1862,12 @@ def test_to_datetime_infer_datetime_format_series_start_with_nans(self, cache):
             pd.to_datetime(s, infer_datetime_format=True, cache=cache),
         )
 
+    def test_infer_datetime_format_tz_name(self):
+        s = pd.Series(["2019-02-02 08:07:13 UTC"])
```

Comment:
is it just UTC that gets recognized?  "GMT"?  "UTC-3"?

Reply:
Recognizes UTC-3 but not GMT. I'll update with a parameterized test.